spec/controllers/application_controller_spec.rb

```diff
@@ -4,14 +4,17 @@
 
 describe ApplicationController do
   describe '#feature_flags' do
+    let(:controller) { described_class.new }
```

Comment:
I think this may be the same as the implicit subject (perhaps also already named controller?)

Reply:
nice catch. Updated accordingly.